Packs/RSANetWitness_v11_1/Integrations/RSANetWitnessv115/RSANetWitnessv115.py

```diff
@@ -1151,6 +1176,231 @@ def get_now_time() -> str | None:
         return None
 
 
+'''Mirror in and out'''
+
+
+def get_mapping_fields_command() -> GetMappingFieldsResponse:
+    mapping_response = GetMappingFieldsResponse()
+    incident_type_scheme = SchemeTypeMapping(type_name='RSA Netwitness incident')
+
+    for field in OUTGOING_MIRRORED_FIELDS:
+        incident_type_scheme.add_field(name=field, description='the description for the field')
+
+    mapping_response.add_scheme_type(incident_type_scheme)
+
+    return mapping_response
+
+
+def xsoar_status_to_rsa_status(xsoar_status: int, xsoar_close_reason: str) -> str:
+    if xsoar_status == 2 and xsoar_close_reason == "False positive":
+        return "ClosedFalsePositive"
+    elif xsoar_status == 2:
+        return "Closed"
+    elif xsoar_status == 1:
+        return "New"
+    return None
+
+
+def update_remote_system_command(client: Client, args: dict, params: dict) -> str:
+    """update-remote-system command: pushes local changes to the remote system
+
+    :type client: ``Client``
+    :param client: XSOAR client to use
+
+    :type args: ``Dict[str, Any]``
+    :param args:
+        all command arguments, usually passed from ``demisto.args()``.
+        ``args['data']`` the data to send to the remote system
+        ``args['entries']`` the entries to send to the remote system
+        ``args['incidentChanged']`` boolean telling us if the local incident indeed changed or not
+        ``args['remoteId']`` the remote incident id
+
+    :return:
+        ``str`` containing the remote incident id - really important if the incident is newly created remotely
+
+    :rtype: ``str``
+    """
+    parsed_args = UpdateRemoteSystemArgs(args)
+    if parsed_args.delta:
+        demisto.debug(f'Got the following delta keys {str(list(parsed_args.delta.keys()))}')
+
+    demisto.debug(f"Starting mirror out for the remote incident {parsed_args.remote_incident_id}")
+    new_incident_id: str = parsed_args.remote_incident_id
+
+    xsoar_status = parsed_args.data.get("status")
+    xsoar_close_reason = parsed_args.data.get("closeReason")
+    response = client.get_incident_request(new_incident_id)
+    rsa_status = xsoar_status_to_rsa_status(xsoar_status, xsoar_close_reason)
+
+    if rsa_status != None and response["status"] != rsa_status:
```

Comment:
The issue with assignee is that if both RSA and XSOAR are not connected to the same LDAP or something similar the XSOAR user may not be the same as RSA and we can have error updating. And since I don't have access to RSA administration user interface to check everything I didn't implement it !
The documentation says: "The assignee field must include the unique identifier for a valid NetWitness user. The list of users can be found in the security section of the administration user interface."

Reply:
Ok, then let's add this limitation to the README.